services/blog.py

```
@@ -389,3 +394,125 @@ def _generate_content(self, digest: Dict[str, Any]) -> str:
+                json=payload,
+                headers={"Content-Type": "application/json"},
+                timeout=60
+            )
+            response.raise_for_status()
+            
+            # Parse the AI response
+            ai_response = response.json()
+            
+            # Validate the response structure
+            if not isinstance(ai_response, dict):
+                raise ValueError("Invalid response format from AI service")
```

Ruff (0.12.2)

Prefer `TypeError` exception for invalid type

(TRY004)